pages/src/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "env": {
+    "browser": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "script"
+  }
+}
```

Comment:
Of course it's safe 👍 check http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy